twitter/View Controllers/TimelineViewController.m

```diff
@@ -47,5 +148,44 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 }
 */
 
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    if ([segue.identifier isEqualToString:@"composeSegue"]) {
```

Comment:
If you have static string constants for segue identifiers, you can use the constant here and where it's invoked to reduce the chance of typos. In your case, composeSegue is coming from the storyboard so having a constant for that one is not too useful, but it could be used for profileSegue.